src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/CommonLanguageServerConstants.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Microsoft.VisualStudio.LanguageServer.ContainedLanguage
+{
+    internal static class CommonLanguageServerConstants
```

Comment:
>CommonLanguageServerConstants [](start = 26, length = 29)

This is minimal copy of LanguageServerConstants from Microsoft.AspNetCore.Razor.LanguageServer.Common to avoid taking dependency on that. Suggestions on how to handle this better are welcome :)

Reply:
Sounds good, I'll just leave it as is for now and we will remove the entire class once we don't need it. 

---
In reply to: [437118181](https://github.com/dotnet/aspnetcore-tooling/pull/1994#discussion_r437118181) [](ancestors = 437118181)